instrumentation-core/spring/spring-webflux-5.0/src/main/java/io/opentelemetry/instrumentation/springwebflux/client/SpringWebfluxHttpClientDecorator.java

```diff
@@ -14,21 +14,18 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.auto.instrumentation.springwebflux.client;
+package io.opentelemetry.instrumentation.springwebflux.client;
 
 import io.opentelemetry.OpenTelemetry;
 import io.opentelemetry.auto.bootstrap.instrumentation.decorator.HttpClientDecorator;
 import io.opentelemetry.trace.Span;
 import io.opentelemetry.trace.Tracer;
 import java.net.URI;
 import java.util.List;
-import lombok.extern.slf4j.Slf4j;
 import org.springframework.web.reactive.function.client.ClientRequest;
 import org.springframework.web.reactive.function.client.ClientResponse;
 
-@Slf4j
-public class SpringWebfluxHttpClientDecorator
-    extends HttpClientDecorator<ClientRequest, ClientResponse> {
+class SpringWebfluxHttpClientDecorator extends HttpClientDecorator<ClientRequest, ClientResponse> {
 
   public static final Tracer TRACER =
```

Comment:
not familiar w/ the tracer provider and the value passed in `get`. Is it something we can inject instead? i.e., `Tracer` should prob be produced as a bean.

Reply:
Nice to meet you here, Ray! :) 

@mabdinur you have done this injection for `HttpServerTracer`, remember? Please do the same for this decorator too. Just add one more constructor here which takes an instance of `Tracer`.

Reply:
I was tried to minimize the changes to the instrumentation code because  I wanted to push out small pull requests and to avoid introducing regressions. @Nikita I will open another pull request to clean up the injection